scripts/apps/search/helpers.tsx

```diff
@@ -133,9 +133,21 @@ export function positionPopup(target, zIndex = 1000) {
     menuHolderElem().style.zIndex = zIndex.toString();
 }
 
-export function renderArea(area: string, itemProps, props?: { className?: string }, customRender: any = {}) {
+interface IItemProps {
+    item: any;
+    listConfig?: any;
+    singleLine?: any;
+    narrow?: any;
+}
+
+export function renderArea(
+    area: string,
```

Comment:
is this only 'firstLine' and 'secondLine'? If so we could change the type of it so it doesn't allow any string

Reply:
Since I already merged this, I'll update the type in some other unrelated PR ;D